dashboard/app/controllers/home_controller.rb

```diff
@@ -46,21 +46,8 @@ def health_check
   # Signed out: redirect to /courses
   def index
     if current_user
-      if current_user.student? && current_user.assigned_course_or_script? && !account_takeover_in_progress? && current_user.primary_script
-
-        # Send students in course experiments (such as the subgoals experiment)
-        # to the right place when they end up on the wrong version of their script.
-        #
-        # In the future, when primary_script selects the script the student is
-        # assigned to rather then where they last made progress, this check can
-        # be removed.
```

Comment:
Given that we're now checking for the script that was last assigned, I think, as the comment suggests, this can be removed. @davidsbailey as the comment writer, is that true?